libs/commons/dune

```diff
@@ -2,6 +2,7 @@
  (public_name commons)
  (wrapped false)
  (libraries
+   digestif.c
```

Comment:
Did you ever get a reponse about why we were using the ocaml version? I we should double check that all the tests go through on the pro-engine before landing this because there are some CI tests that use an ocaml to js backend that could choke it they are using this.

Reply:
Hmm, I had changed this back---may just have not pushed that commit. Not needed since we aren't using `aws` directly, but that would be another blocker even if the upstream function worked.